kolibri/core/discovery/test/test_network_utils.py

```diff
@@ -301,6 +302,14 @@ def test_request__response_failure(self):
                     with NetworkClient("http://sadurl.qqq") as nc:
                         nc.connect()
 
+    @mock.patch.object(requests.Session, "get", mock_response(200))
+    def test_request__user_agent(self):
+        client = NetworkClient()
+        response = client.get("https://example.com")
```

Comment:
This test is failing because `NetworkClient` requires an argument `base_url`.

```suggestion
        client = NetworkClient("https://example.com")
        response = client.get("/")
```